CMakeLists.txt

```diff
@@ -2087,6 +2087,13 @@ include(HPX_SetupBoost)
 include(HPX_SetupBoostFilesystem)
 include(HPX_SetupBoostIostreams)
 include(HPX_SetupBoostProgramOptions)
+
+if("${__target_arch}" STREQUAL "ppc" OR "${__target_arch}" STREQUAL "ppc64")
+  if ("${Boost_VERSION_STRING}" VERSION_LESS "1.69.0")
+    message(FATAL_ERROR "Boost versions < 1.69.0 are unsupported on PPC architecture")
```

Comment:
Please use `hpx_error("...")` istead

Reply:
Also, wouldn't it be better to have this check in https://github.com/STEllAR-GROUP/hpx/blob/master/cmake/HPX_SetupBoost.cmake?

Reply:
I'll move it to setup boost - the reason I didn't put it there is because the var  `__target_arch` doesn't exist in that file and introducing it there seemed wrong. I'll just add a comment to say that it is set already in the main cmakelists file.